### DISCUSSION TITLE
Problems with MLP_run notebook: always only eight compounds forecasted as RNA_binder

### DISCUSSION THREAD

**Discussion by agiani99** [ky66/ROBIN#4] [Q&A] [answered]
Hi everyone,
Sorry to bug you with this, but I wonder whether it could be a possible bug. 
RNA_binder prediction notebook. Independently from number of molecules presented in the library to be predicted, every time only eight compounds seems to be found as binders. is it possible? Why all the rest of library has no probabilities in predictions? Try with the attached file.
Best
Andrea
[test_3D_desc.csv](https://github.com/ky66/ROBIN/files/9331397/test_3D_desc.csv)

**Comment by ky66** [comment, marked as answer]
Hey Andrea! 
I just looked at your CSV file of generated Mordred chemical features. I can see that 3D features starting with Mor such as for instance (Mor26se) are all empty which is causing problems in the model. This problem is often caused by your starting SDF file often not being 3D. I would take your starting 2D SDF file into DataWarrior and generate the 3D conformers as described in the Readme page.

You are right about the bug with the nan removal only applying to the first 8 compounds. That is been fixed in the new push. Thanks for pointing this out!

I hope this helps! Please let us know if you have any other questions. Thanks
Best,
Kamyar
